torch/fx/subgraph_rewriter.py

```diff
@@ -60,25 +63,52 @@ def attributes_are_equal(pn : Node, gn : Node) -> bool:
             return False
         match_found = all(self._match_nodes(pn_, gn_) for pn_, gn_
                           in zip(pn.all_input_nodes, gn.all_input_nodes))
+        match_found = False
+        if pn.op == "output":
```

Comment:
What I meant by the comment on the other PR was that going through permutations should be unnecessary so long as we are enforcing that the pattern graph only has one output value. In that case, I think this section should be something like:
```
match_found = any(self._match_nodes(pn_.all_input_nodes[0], gn_) for gn_ in gn.all_input_nodes)
```

Reply:
Now that this is the logic, you'll want the fact that the pattern only has 1 output enforced. That's in the other PR here https://github.com/pytorch/pytorch/pull/51412/files#diff-14d3157db8fc26c5c1c0f18e780290a2b527c468dfc63cb3b64aa886c078502eR21 so if you land that one before this one (or put them in a stack) it should be fine